### DISCUSSION TITLE
HowTo achieve an automatic synchronization

### DISCUSSION THREAD

**Discussion by pwsimon** [dexie/Dexie.js#1855] [Q&A] [answered]
when i add a row to a table using the [Cloud REST API](https://dexie.org/cloud/docs/rest-api#post-and-delete) my Application will not automatically synchronized.
after performing a [db.cloud.sync({ purpose: "pull" })](https://dexie.org/cloud/docs/db.cloud.sync()) it is synchronized.
HowTo achieve an automatic synchronization?

**Comment by dfahlander** [comment]
Similar to #1854 but this concerns data and not the user object so it's not a duplicate. I'll convert this one to an issue to track when it's solved.

Issue #1856 created.

**Comment by dfahlander** [comment]
@pwsimon The issue has been fixed in staging environment. If I get the ID of your database I could move it over to staging temporarily and let you verify the fix?

**Reply by pwsimon** [reply]
great offer, let's try it `https://zupdaunsr.dexie.cloud`

**Comment by dfahlander** [comment, marked as answer]
Thanks! It's lifted to staging right now. Please let me know with a thumbs up if it works as expected. When this version is in production I will move your db back to prod again. No data has been moved - it's just the node backend code using a newer version.
